public/enhanced-table-vis-controller.js

```diff
@@ -308,12 +308,30 @@ module.controller('EnhancedTableVisController', function ($scope, Private, confi
 
       let removedCounter = 0;
       _.forEach(hiddenColumns, function (item) {
-        let index = getRealColIndex(parseInt(item), splitColIndex);
-        table.columns.splice(index - removedCounter, 1);
-        _.forEach(table.rows, function (row) {
-          row.splice(index - removedCounter, 1);
-        });
-        removedCounter++;
+        try {
+          if (parseInt(item).toString() == item) {
+            let index = getRealColIndex(parseInt(item), splitColIndex);
+            let colToRemove = index - removedCounter;
+            console.log(colToRemove)
```

Comment:
I understand that you use console.log during debug, but please remove them before publishing code.

Reply:
So sorry man. I pushed in a rush at work today. That's why code revision is fundamental :).
I'll remove all the console.logs in the next commit